<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PhaseWPHabit1</class>
 <widget class="QWizardPage" name="PhaseWPHabit1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <property name="title">
   <string>Reduction of looking time</string>
  </property>
  <property name="subTitle">
   <string>Enter habituation criteria details</string>
  </property>
  <attribute name="pageId">
   <string notr="true">4</string>
  </attribute>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QRadioButton" name="rbUseFirstWindow">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Habit will use total looking time from the &lt;span style=&quot; font-style:italic;&quot;&gt;first full window&lt;/span&gt; as the basis for habituation.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>Use first window</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rbUseMaxWindow">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Habit will use the window with the &lt;span style=&quot; font-style:italic;&quot;&gt;most looking time&lt;/span&gt; as the basis for habituation.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>Use max window</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Choose the method used for determining the basis for habituation. Total looking in the basis window will be used, along with the settings below, to determine when habituation is achieved. </string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QCheckBox" name="cbxRequireMinBasisValue">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Require a &lt;span style=&quot; font-style:italic;&quot;&gt;minimum amount of looking &lt;/span&gt;for a basis window. If &lt;span style=&quot; font-style:italic;&quot;&gt;first window&lt;/span&gt; is selected, then the first window &lt;span style=&quot; font-style:italic;&quot;&gt;which passes this test&lt;/span&gt; will be used as the basis.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>Require minimum basis value</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineeditMinBasisValue">
          <property name="text">
           <string>10000</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>ms</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Percentage</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinPercentage">
          <property name="value">
           <number>50</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Habituation is achieved if the sum of looking in a window &lt;span style=&quot; font-style:italic;&quot;&gt;falls to this percentage (or less)&lt;/span&gt; of the sum of looking in the basis window.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="cbxExcludeBasisWindow">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;When using a &lt;span style=&quot; font-style:italic;&quot;&gt;sliding window&lt;/span&gt;, the trials used as the basis are not used when computing percentage and testing for habituation. This has no effect when using a &lt;span style=&quot; font-style:italic;&quot;&gt;fixed window.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Exclude basis window when computing percentage</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_3">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Window Type:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="comboWindowType">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;For a &lt;span style=&quot; font-style:italic;&quot;&gt;fixed window, &lt;/span&gt;windows are [&lt;span style=&quot; font-style:italic;&quot;&gt;1,2,...,size], [size+1,...,2*size],...&lt;/span&gt;&lt;/p&gt;&lt;p&gt;For a &lt;span style=&quot; font-style:italic;&quot;&gt;sliding window, &lt;/span&gt;windows are &lt;span style=&quot; font-style:italic;&quot;&gt;[1,2,...size], [2,3,...,size+1],...&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Window size (trials):</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="spinWindowSize">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="value">
           <number>3</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Specify your definition of a &lt;span style=&quot; font-style:italic;&quot;&gt;window. &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
